src/shared/tpm2-util.c

```diff
@@ -2354,14 +2356,20 @@ int tpm2_pcr_mask_from_string(const char *arg, uint32_t *ret_mask) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to parse PCR list: %s", arg);
 
+                if (isalpha(pcr[0])) {
+                r = named_pcr(pcr, &n);
```

Comment:
indentation is not right, see docs/CODING_STYLE.md

Reply:
> indentation is not right, see docs/CODING_STYLE.md

Fixing it immediately.